scripts/start.js

```diff
@@ -365,10 +376,29 @@ function getUserInfo () {
         return reject(err)
       }
       if (body['access_token']) {
-        resolve({
-          userName: username.slice(0, username.indexOf('@')),
-          ssoToken: body.access_token,
-          domain: domain
+        request(`${engineUrl}api/users`, { json: true, strictSSL: false, headers: { Authorization: `Bearer ${body['access_token']}` } }, (err2, response2, body2) => {
```

Comment:
Could we please use more self-explanatory names like `usersErr`, `usersResponse`, `usersBody` instead of indexing?